contracts/strategies/auraBAL/rewardHandlers/BBUSDHandler.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "./HandlerBase.sol";
+
+contract BBUSDHandler is HandlerBase {
+    using SafeERC20 for IERC20;
+
+    address private constant BBUSD_TOKEN =
+        0x7B50775383d3D6f0215A8F290f2C9e2eEBBEceb2;
+    address private constant BBUSDC_TOKEN =
+        0x9210F1204b5a24742Eba12f710636D76240dF3d0;
+    address private constant USDC_TOKEN =
+        0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+
+    bytes32 private constant BBUSD_AAVE_POOL_ID =
+        0x7b50775383d3d6f0215a8f290f2c9e2eebbeceb20000000000000000000000fe;
+    bytes32 private constant BBUSDC_USDC_POOL_ID =
+        0x9210f1204b5a24742eba12f710636d76240df3d00000000000000000000000fc;
+    bytes32 private constant ETH_USDC_POOL_ID =
+        0x96646936b91d6b9d7d0c47c496afbf3d6ec7b6f8000200000000000000000019;
+
+    constructor(address _token, address _strategy)
+        HandlerBase(_token, _strategy)
+    {}
+
```

Comment:
For `AuraHandler.sol` you have a `setApprovals()`, but don't you need one for `BBUSDHandler.sol` as well so `balVault` can swap `BBUSD_TOKEN` for `WETH` in the batch swap?